File: backend/WebApi/Lunchorder.Domain/Entities/DocumentDb/LastOrderEntry.cs
using System;

namespace Lunchorder.Domain.Entities.DocumentDb
{
    public class LastOrderEntry
    {
        /// <summary>
        /// The order name
        /// </summary>
        public string Name { get; set; }
        /// <summary>
        /// The rules that were applied (one per line)
        /// </summary>
        public string AppliedRules { get; set; }

        /// <summary>
        /// The freetext that a user entered
        /// </summary>
        public string FreeText { get; set; }

        /// <summary>
        /// The price for the entry
        /// </summary>
        public decimal Price { get; set; }
    }
}

File: frontend/src/app/domain/dto/lastOrderEntry.ts
export class LastOrderEntry implements app.domain.dto.ILastOrderEntry, Serializable<LastOrderEntry> {
        name: string;
        appliedRules: string;
        freeText: string;
        price: number;

        deserialize(input: any) : LastOrderEntry {
                if (!input) return;
                this.name = input.name;
                this.appliedRules = input.appliedRules;
                this.freeText = input.freeText;
                this.price = input.price;
                return this;
        }
}